src/store/ui/useStudioUIStore.ts

```diff
@@ -1,18 +1,21 @@
+import { SettingsTabValue } from '@/components/Settings/types'
 import { FolderContent, ProxyStatus, StudioFile } from '@/types'
 import { create } from 'zustand'
 import { immer } from 'zustand/middleware/immer'
 
 interface State extends FolderContent {
   proxyStatus: ProxyStatus
   isSettingsDialogOpen: boolean
+  selectedSettingsTab: SettingsTabValue
 }
 
 interface Actions {
   addFile: (file: StudioFile) => void
   removeFile: (file: StudioFile) => void
   setFolderContent: (content: FolderContent) => void
   setProxyStatus: (status: ProxyStatus) => void
-  setIsSettingsDialogOpen: (isOpen: boolean) => void
+  openSettingsDialog: (tab?: SettingsTabValue) => void
+  closeSettingsDialog: () => void
```

Comment:
I decomposed `setIsSettingsDialogOpen` into separate methods which makes open/close more explicit. It allows (optionally) passing a specific tab only when `openSettingsDialog` is called.